drivers/gpio/Kconfig.pcal64xxa

```diff
@@ -0,0 +1,36 @@
+# PCAL6408a GPIO configuration options
+
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2023 SILA Embedded Solutions GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig GPIO_PCAL64XXA
+	bool "PCAL64XXA I2C GPIO chip"
+	default n
+	help
+	  Enable driver for PCAL64XXA I2C GPIO chip.
+
+config GPIO_PCAL64XXA_INIT_PRIORITY
+	int "Init priority"
+	default 70
+	depends on GPIO_PCAL64XXA
+	help
+	  Device driver initialization priority.
+
+config GPIO_PCAL6408A
+	bool "PCAL6408A I2C GPIO chip"
+	default y
+	depends on DT_HAS_NXP_PCAL6408A_ENABLED
+	depends on I2C
+	select GPIO_PCAL64XXA
+	help
+	  Enable driver for PCAL6408A I2C GPIO chip.
+
+config GPIO_PCAL6416A
+	bool "PCAL6416A I2C GPIO chip"
+	default y
+	depends on DT_HAS_NXP_PCAL6416A_ENABLED
+	depends on I2C
+	select GPIO_PCAL64XXA
+	help
+	  Enable driver for PCAL6416A I2C GPIO chip.
```

Comment:
You need to loose the chip specific Kconfigs. See https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/adc/Kconfig.lmp90xxx and https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/adc/Kconfig.mcp320x for inspiration.

Reply:
done